app/pages/project/talk.cjsx

```diff
@@ -9,13 +9,21 @@ TalkFootnote = require '../../talk/footnote'
 module.exports = React.createClass
   displayName: 'ProjectTalkPage'
 
+  contextTypes:
+    geordi: React.PropTypes.object
+
+  componentWillMount: ->
+    @context.geordi?.logEvent
```

Comment:
They look a bit different because one is on a click while the other executes with the page load. They could both probably be abstracted into the same function though.